### DISCUSSION TITLE
Is that possible to restart a WinUI app from a mornal privilege to elevated?

### DISCUSSION THREAD

**Discussion by Lightczx** [microsoft/WindowsAppSDK#3453] [Q&A]
As far as I know, `AppInstance.Restart` can restart from normal to normal or elevated to elevated,but how can I restart from normal to elevated?

**Comment by DarranRowe** [comment]
If it is unpackaged then [ShellExecute](https://learn.microsoft.com/en-us/windows/win32/api/shellapi/nf-shellapi-shellexecutew) with the runas verb will do it. If it is packaged, then I think you need the [full trust process launcher](https://learn.microsoft.com/en-us/uwp/api/Windows.ApplicationModel.FullTrustProcessLauncher?view=winrt-22621) and the runFullTrust restricted capability.

**Reply by Lightczx** [reply]
I'm creating a packaged (single project) WinUI3 app, how can I use the FullTrustProcessLauncher to launch the app self?

**Reply by Lightczx** [reply]
Oh I see, I miss some method in the class
